spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -51,16 +67,50 @@ def self.from_territories(territories = [], params = {})
         obj.include_future_territories = params.fetch(:include_future_territories, true)
         obj.cleared_for_preorder = params.fetch(:cleared_for_preorder, false)
         obj.app_available_date = params.fetch(:app_available_date, nil)
+        obj.b2b_unavailable =  params.fetch(:b2b_unavailable, false)
+        obj.b2b_app_enabled =  params.fetch(:b2b_app_enabled, false)
+        obj.educational_discount = params.fetch(:educational_discount, true)
         return obj
       end
 
       def territories
         @territories ||= raw_data['countries'].map { |info| Territory.new(info) }
       end
 
+      def b2b_users
+        @b2b_users ||= raw_data['b2bUsers'].map { |user| B2bUser.new(user) }
+      end
+
+      def b2b_app_enabled
+        @b2b_app_enabled.nil? ? raw_data['b2bAppEnabled'] : @b2b_app_enabled
+      end
+
+      def educational_discount
+        @educational_discount.nil? ? raw_data['educationalDiscount'] : @educational_discount
+      end
+
       def cleared_for_preorder
         super || false
       end
+
+      # Sets the b2b flag. If you call Save on app_details without adding any b2b users
```

Comment:
Can we change this comment to...
```rb
# Sets `b2b_app_enabled` to true and `educational_discount` to false
# Requires users to be added with `add_b2b_users` otherwise `update_availability` will error
```